#!/usr/bin/python3
def raise_exception_msg(message=""):
    """Raises a NameError with a message"""
    raise NameError(message)

